where-to-travel/src/main/webapp/script.js

```diff
@@ -80,6 +81,18 @@ script.async = true;
 
 document.head.appendChild(script);
 
+$('.multi-select-pill').click(function () {
+  let pillText = $(this).text();
+  if(pillText === selectedPillText) {
+    placeType = '';
```

Comment:
Am I understanding correctly that in this case, they've clicked on the already selected pill so now there would be no selected type? Just curious.

Reply:
Yes, if they click on the same selected placeType, then it sets the placeType to empty